charts/konk/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
Oh, we haven't done this with any of the others yet, but we need to start adding `metadata.namespace` to all of the resources so they land in the right namespace even when Spinnaker doesn't override the namespace.

Reply:
Added Namespace info to all the resources.